src/@awesome-cordova-plugins/plugins/airship/index.ts

```diff
@@ -0,0 +1,1755 @@
+import { Injectable } from '@angular/core';
+import {
+  Plugin,
+  Cordova,
+  AwesomeCordovaNativePlugin,
+  CordovaInstance,
+  InstanceProperty,
+  CordovaCheck,
+} from '@awesome-cordova-plugins/core';
+import { Observable, Observer } from 'rxjs';
+
+declare const window: Window & { Airship: any };
+
+/**
+ * @name AirShip
+ * @description
+ * This plugin does something
+ * @usage
+ * ```typescript
+ * import { AirShip } from '@awesome-cordova-plugins/airship/ngx';
+ *
+ * constructor(private airShip: AirShip) { }
+ *
+ * ...
+ *
+ * this.platform.ready().then(() => {
+ *    this.airShip.initialize();
+ *    this.airShip.setUserNotificationsEnabled(true)
+ *      .then((result: any) => console.log(res))
+ *      .catch((error: any) => console.error(error));
+ * })
+ *
+ * ```
+ */
+@Plugin({
+  pluginName: 'AirShip',
+  plugin: '@ua/cordova-airship',
+  pluginRef: 'Airship',
+  repo: 'https://www.npmjs.com/package/@ua/cordova-airship',
+  install: 'ionic cordova plugin add @ua/cordova-airship',
+  platforms: ['Android', 'iOS'],
+})
+@Injectable()
+export class AirShip extends AwesomeCordovaNativePlugin {
```

Comment:
If we are doing a breaking change this should be `Airship` or does that conflict with the cordova name?

Reply:
changed to `Airship`